apps/web/src/components/forms/profile.tsx

```diff
@@ -44,7 +44,7 @@ export const ProfileForm = ({ className, user }: ProfileFormProps) => {
   } = useForm<TProfileFormSchema>({
     values: {
       name: user.name ?? '',
-      signature: '',
+      signature: user.signature ? user.signature : '',
```

Comment:
`user.signature || ''` should work, my bad on this one I figured it might auto-populate based on the default value.